lib/rentlinx/client.rb

```diff
@@ -2,9 +2,18 @@
 require 'json'
 require 'uri'
 require 'rentlinx/default'
+require 'rentlinx/modules/property_client_methods'
+require 'rentlinx/modules/unit_client_methods'
+require 'rentlinx/modules/photo_client_methods'
+require 'rentlinx/modules/amenity_client_methods'
 
 module Rentlinx
   class Client
+    include Rentlinx::PropertyClientMethods
```

Comment:
Pulling out these modules is a nice step towards better separation of concerns, but I think we might want to go a little farther and separate these sets of functionality into independent classes.  It's a little funny to me that that the modules define private methods that are only called in the class they're being included into - to me it makes the interface a bit harder to understand.  This class is also pretty large when you consider all the code that's being included into it, and has several different responsibilities.  As a side benefit, we could probably pull out some kind of base class or module that could be shared by photo_client_methods and amenity_client_methods since there's a lot of duplicated logic between them (we could do that without making them separate classes, but it'd be a bit easier if they were).  Anyway, it may not be worth doing now, but it's something to think about.